.github/workflows/e2e.yml

```diff
@@ -0,0 +1,45 @@
+name: Run e2e tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    name: Run e2e tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Use pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 6.0.2
+
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: E2E script
+        id: e2e-run
+        run: node e2e.js
+
+      - name: Report failure
+        if: ${{ failure() && github.repository_owner == 'nrwl' && github.event_name != 'workflow_dispatch' }}
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: 'failure'
+          message_format: '{emoji} Workflow has {status_message} ${{ steps.e2e-run.outcome }}'
+          notification_title: '{workflow}'
+          footer: '<{run_url}|View Run> / Last commit <{commit_url}|{commit_sha}>'
+          mention_groups: '@Isaac'
```

Comment:
do you want to expand this potentially? So more ppl see it? Could go to the Nx docs/content team that Craigory has set up also for reviewing nx.dev docs

Reply:
This is a slack message, and the docs team is defined in github.  Once I figure out the authentication piece, we can fine tune how gets notified and where the message is posted.